### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by greggman** [KokoDoko/PacmanLamp#1]
This project is amazing. I used your code to build a lamp and it worked great! Now I'm adding new things and I wanted to post the code on another github repo (of course linking to this) but I can't do that without knowing the license. Can you add one? Thank you.

**Comment by KokoDoko** [comment]
Hope this CC non commercial license is ok for your purposes! It's cool to see what others do with the project!

**Comment by greggman** [comment]
Thanks. That works. Although FYI. The Creative Commons site says not to use Creative Commons for code

https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software

**Comment by greggman** [comment]
So thank you for the inspiration. I ended up just re-writing from scratch as I decided I didn't want to CC-NC and it would only take a short time to reproduce from scratch. But no worries.

If you're curious [here's the project](https://www.youtube.com/watch?v=5vNteZiAiWw) and while that video has your pacman code in it, the rewrite from scratch does a few minor different things. The ghosts go at different speeds when chasing pacman. And pacman eats the ghosts. [Here's a video](https://www.youtube.com/watch?v=r5xicvsHEbI)
